src/handlers/dataProvidersHandlers.ts

```diff
@@ -98,6 +113,7 @@ const putDataProvider = (
 		const dataProvider = await createOrUpdateDataProvider({
 			shortCode,
 			name,
+			keycloakOrganizationId,
```

Comment:
This is explicitly setting the nullable key, as opposed to omitting the key entirely when it is `null`.

Reply:
I think this note was intended as a hint regarding an earlier note about undefined vs null. I don't think anything else is needed here, right?